### DISCUSSION TITLE
Bitme

### DISCUSSION THREAD

**Discussion by Getze56901** [iptv-org/community#713] [Q&A]
Hello This is a request for add this channel about videogames and anime in mexico and latam 

Channel Name: Bitme
Channel ID: BitMe.mx
Country: Mexico
Language: Spanish
Website: https://www.bitme.gg/

**Comment by BellezaEmporium** [comment]
Will be difficult since there's no Internet stream, Televisa chose to rely on Satellite + ISP offers for this channel.

I found BitMe on a Sky México pack https://www.lyngsat.com/muxes/Sky-Mexico-1_Mexico_12060-V.html , apart from the official website this is the only info I have.

**Comment by Sphinxroot** [comment]
https://megatelevisionhd.com/en-vivo/bitme/

mpd dash

**Comment by BellezaEmporium** [comment]
https://live4-ott.izzigo.tv/out/u/dash/BIT-ME-HD/default.mpd (Mexico Only - Widevine - ?)
https://live4-ott.izzigo.tv/out/u/hls/BIT-ME-HD/default.m3u8 (Mexico Only, tested with VPN)

https://channel-tiin-cdn.blim.com/manifest.mpd (Worldwide - Widevine - https://api.blim.com/license/widevine)
https://channel-tiin-cdn.blim.com/manifest.m3u8

**Comment by freearhey** [comment]
https://live4-ott.izzigo.tv/out/u/hls/BIT-ME-HD/default.m3u8
<img width="649" alt="image" src="https://user-images.githubusercontent.com/7253922/185200226-72b1e2de-729d-4486-831f-afad3ff026fa.png">

https://channel-tiin-cdn.blim.com/manifest.m3u8
not loading

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/mx.m3u#L30-L31
